tools/chocolateyinstall.ps1

```diff
@@ -2,14 +2,14 @@
 $ErrorActionPreference = 'Stop'; # stop on all errors
 $toolsDir   = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
 
-$url64      = 'https://github.com/wasmCloud/wash/releases/download/v0.12.0/wash.exe'
+$url64      = 'https://github.com/wasmCloud/wash/releases/download/v0.13.0/wash.exe'
 
 $packageArgs = @{
   packageName   = $env:ChocolateyPackageName
   fileFullPath  = "$toolsDir\wash.exe"
   url64bit      = $url64
 
-  checksum64    = '770E564990B075CF714E000559290BBF9F774C7C6360261B4499C74F2C648641'
+  checksum64    = 'b8bacfbfdbaa84aa35be16a7fb0cac911fa28878cd32c5cb93fc4de6a3a2650f'

```

Comment:
This, annoyingly enough, has to be uppercase

Reply:
WINDOWSSSSSSSSS!!!!!!!!!!